alacritty/src/cli.rs

```diff
@@ -72,6 +72,10 @@ pub struct Options {
     /// Subcommand passed to the CLI.
     #[clap(subcommand)]
     pub subcommands: Option<Subcommands>,
+
+    /// Use EGL as display API if the current platform allows it.
+    #[clap(long)]
+    pub prefer_egl: bool,
```

Comment:
We don't special options like that for debug settings, they could be set via the `-o`.